test/e2e/csv_e2e_test.go

```diff
@@ -3315,219 +3316,1048 @@ var _ = Describe("CSV", func() {
 
 		require.Equal(GinkgoT(), apiService1.Spec.CABundle, apiService2.Spec.CABundle)
 	})
-})
+	It("creates global webhooks", func() {
```

Comment:
this can be a follow up, but we're trying to write new tests in a more BDD style, check out harish's PR: https://github.com/operator-framework/operator-lifecycle-manager/pull/1424

The tests look good otherwise, just a style thing.

Reply:
Noted.